src/Select/Select.less

```diff
@@ -75,9 +79,10 @@
 
       .Select-input,
       .Select-value {
-        font-size: 1rem;
-        line-height: 40px;
-        top: 10px;
+        .padding--left--s;
+        .text--medium;
+        line-height: @size_2xl;
+        top: @size_s;
```

Comment:
`.padding--left--s = padding-left: 12px` Added to align with TextInput ([analogous value](https://github.com/Clever/components/blob/2bb27f80302a71cc7ccff822a38d15fb2304b653/src/TextInput/TextInput.less#L9))
`.text--medium = font-size: 1rem`
`@size_2xl = 40px`
`@size_s = 12px` Larger than the original value but identical to TextInput's [analogous value](https://github.com/Clever/components/blob/2bb27f80302a71cc7ccff822a38d15fb2304b653/src/TextInput/TextInput.less#L52)